pepper-apis/dss-core/src/main/resources/logback.xml

```diff
@@ -4,7 +4,7 @@
         <!-- this filter is here to help prevent sensitive data like PHI from bleeding into stdout -->
         <filter class="org.broadinstitute.ddp.logging.NonSecureFilter" />
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} %X{LogBreadCrumb} %X{X-Forwarded-For} C:%X{ClientId} S:%X{UserId} [%thread] %-5level %logger{36} %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %X{LogBreadCrumb} %X{X-Forwarded-For} C:%X{ClientId} S:%X{UserId} [%thread] %-5level %logger{36} %msg D:%X{GAE_DEPLOYMENT_ID} I:%X{GAE_INSTANCE}%n</pattern>
```

Comment:
Adding in support for PEPPER-1412 as well.  Logback line was getting pretty busy on the left side, so I added these GAE values on the right side of the log message.